tunnel/intra/direct_split.go

```diff
@@ -0,0 +1,69 @@
+package intra
+
+import (
+	"io"
+	"net"
+)
+
+type splitter struct {
+	DuplexConn
+	conn  *net.TCPConn
+	fresh bool
```

Comment:
This variable tracks whether we have read from/written to this connection, right? Consider renaming or adding a comment.

Reply:
Done.  (Also this was initialized to false!)